requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Core packages needed
-Django>=3.2, <4.0
+Django>=3.2, <6.0
```

Comment:
Is there a reason this is 6.0 as opposed to the highest supported version?

Reply:
The highest version currently is 5.0. I want to make the package ready for the 5.0 upgrade, so the next person that needs to update Django doesn't have any issues.